locales/hr-HR.json

```diff
@@ -0,0 +1,10 @@
+[
+  "upravo sad",
+  ["prije %s sekundi", "prije %s sekundi"],
```

Comment:
If there's no difference in singular and plural nouns, you can directly use a string instead of array, eg: `'prije %s sekundi'` here